docs/content/docs/install/manual_demo/_index.md

```diff
@@ -327,6 +327,8 @@ The counter in the `bookthief` window will start incrementing.
 
 We will now demonstrate how to balance traffic between two Kubernetes services, commonly known as a traffic split. We will be splitting the traffic directed to the root `bookstore` service between the backends `bookstore-v1` service and `bookstore-v2` service.
 
+_Note:_ The root Kubernetes service should not have any label selectors defined that match a/any pod on the mesh. This may lead to unintended behavior.
+
```

Comment:
@michelleN wouldn't this be the case if we split traffic to the root service to itself and another service, as we discussed offline?

Reply:
yes actually. Going to close out this PR and let #2654 take care of the demo updates.